testing/misc_tools.c

```diff
@@ -35,6 +35,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#ifndef VANILLA_NACL
```

Comment:
You need to `#include "config.h"` like in the other files for the VANILLA_NACL symbol to be available.